tests/php/sync/test_class.jetpack-sync-comments.php

```diff
@@ -47,6 +47,47 @@ public function test_update_comment() {
 		$this->assertEquals( "foo bar baz", $remote_comment->comment_content );
 	}
 
+	public function test_unapprove_comment() {
+
+		$this->assertEquals( 1, $this->server_replica_storage->comment_count( 'approve' ) );
```

Comment:
It would be good to use space to group the different tests that are happening here a bit better.